docs/app/content-collections.md

```diff
@@ -1,168 +1,113 @@
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to filter
-3. Click on Filter (filter icon) located in the header
-4. Click the "Add Filter" dropdown and select the field to filter on
-5. Adjust the operator (eg: "Contains") as needed by clicking on it
-6. Enter a value in the field filter's input(s)
-7. Remove unwanted filters by hovering over the field and clicking "X"
+1. Navigate to the Content Module and select the desired Collection.
+2. In the Sidebar, choose a [Layout](/app/content-layouts/#adjust-item-layouts) that supports ordering.\
+   (e.g. "table" or "card" mode)
+3. Set the Order Field and Order Direction.
+   - **Table Layout:** Click on the Field's column Header, clicking again changes order direction.
+   - **Card Layout:** Adjust the Sort Field and Sort Direction in the Layout Subheader.
 
-## Viewing Archived Items
+::: tip
 
-**[Learn more about Archive](/configuration/data-model/#archive).**
+Ordering is a non-destructive action which does not change your data.
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to view
-3. Click "Archive" in the page sidebar
-4. Choose the desired view: `Show Items` (default), `Show Archived Items`, or `Show Items + Archived Items`
+:::
 
-## Bookmarking Item Presets
+## View Archived Items
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to bookmark
-3. Configure the page as desired (eg: layout, options, search, filters, order, etc)
-4. Click the "Create Bookmark" (bookmark icon) button in the header beside the page title
-5. Rename bookmarks by right-clicking them in the navigation sidebar and selecting "Rename Bookmark"
-6. Delete bookmarks by right-clicking them in the navigation sidebar and selecting "Delete Bookmark"
+![View Archived Items](image.webp)
 
-## Importing / Exporting Items
+An Item can be [archived](/app/content-items/#archive-an-item) so that it exists within the database but is not
+displayed with other "active" Items in the app. To view an archived Item, follow the steps below.
 
-### Import
+1. Navigate to the Content Module and select the desired Collection.
+2. Click **"Archive"** in the Sidebar.
+3. Choose the desired view: `Show Items` (default), `Show Archived Items`, or `Show Items + Archived Items`.
 
-**[Learn more about Import Data from a File](/reference/system/utilities/#import-data-from-file)**
+## Bookmark Item Presets
 
-1. Navigate to the [Content Module](/app/overview/#_1-module-bar)
-2. Navigate to the collection of the items you want to import
-3. Click "Import / Export" in the page sidebar
-4. Choose the desired file containing items to import in the accepted format: either CSV or JSON
-5. Click the "Start Import" button to import the items from the chosen file
+![Bookmark Item Presets](image.webp)
```

Comment:
Requires going into settings to thoroughly explain this. 
Use Video, not image.

Reply:
Exists in config file: Add a link to config file